apps/music/test/unit/metadata/flac_test.js

```diff
@@ -36,6 +45,7 @@ suite('flac tags', function() {
         assert.strictEqual(metadata.trackcount, 8);
         assert.strictEqual(metadata.discnum, 1);
         assert.strictEqual(metadata.disccount, 1);
+//        assert.deepEqual(metadata.picture, expectedPicture);
```

Comment:
Actually I should leave FLAC support out as the test doesn't work. Looking at it closely it seems that the sample store the album art differently that vorbis comment. I think I'll just do that. Split it out, get Ogg support first. *sigh*